packages/web/components/Dashboard/XP.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Stat,
+  StatArrow,
+  StatGroup,
+  StatHelpText,
+  StatLabel,
+  StatNumber,
+} from '@metafam/ds';
+import { useUser } from 'lib/hooks';
+import { useUserXp } from 'lib/hooks/useUserXp';
+import React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
remove this

Reply:
tried but it displayed Type 'Element' is not assignable to type 'FC<{}>'.

Reply:
ok found another type